cumulusci/tasks/release_notes/parser.py

```diff
@@ -295,16 +295,50 @@ def parse(self, change_note):
 
     def render(self, existing_content=""):
         version_id = self.release_notes_generator.version_id
+        trial_info = self.release_notes_generator.trial_info
+
+        if (
+            not version_id
+            and not self.release_notes_generator.sandbox_date
+            and not self.release_notes_generator.production_date
+            and not trial_info
+        ):
+            return existing_content
+        result = [self._render_header()]
+        if (
+            self.release_notes_generator.sandbox_date
+            or self.release_notes_generator.production_date
+        ):
+            result.append("## Push Schedule")
+            if self.release_notes_generator.sandbox_date:
+                result.append(
+                    f"Sandbox orgs: {self.release_notes_generator.sandbox_date}"
+                )
+            if self.release_notes_generator.production_date:
+                result.append(
+                    f"Production orgs: {self.release_notes_generator.production_date}",
+                )
         if version_id:
             version_id = urllib.parse.quote_plus(version_id)
-            return "\r\n".join(
-                [
-                    self._render_header(),
-                    "Production & Developer Edition Orgs:",
-                    f"{PROD_LOGIN_URL}/packaging/installPackage.apexp?p0={version_id}",
-                    "",
-                    "Sandbox & Scratch Orgs:",
-                    f"{SANDBOX_LOGIN_URL}/packaging/installPackage.apexp?p0={version_id}",
-                ]
-            )
-        return existing_content
+            if (
+                self.release_notes_generator.sandbox_date
+                or self.release_notes_generator.production_date
+            ):
+                result.append("")
```

Comment:
Rather than conditionally adding these empty lines at the start of the next subsection, I'd add them at the end of each subsection, without a condition. There's no harm to having an extra blank line at the end.

Reply:
Oh for some reason I wasn't a fan of the way it looked tbh.

Reply:
This was intentional butt will do.